xml/art_suseconnect_tracking.xml

```diff
@@ -0,0 +1,405 @@
+   </dm:bugtracker>
+   <dm:editurl>https://github.com/SUSE/doc-unversioned/blob/main/xml/</dm:editurl>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+  <abstract>
+   <para>
+    This document document describes what &suseconnect; is and how the &scc; (SCC) and Repository
+    Mirroring Tool (RMT) use it to gather information about active systems and their hardware
+    environment.
+   </para>
+  </abstract>
+ </info>
+ <sect1 xml:id="suseconnect-whatis">
+  <title>What is &suseconnect;?</title>
+
+  <para>
+   &suseconnect; is a tool to register SLE-based operating systems with SCC or RMT. Registered
+   systems receive updates to installed packages, for example, to increase security of the operating
+   system. With &suseconnect;, you can additionally register extensions and modules that extend the
+   base system functionality.
+  </para>
+
+  <tip>
+   <para>
+    This document does not cover general usage of &suseconnect;. To find more basic information
+    about how to register your system, modules, and extensions with &suseconnect;, refer to
+    <link xlink:href="https://documentation.suse.com/sles/html/SLES-all/cha-register-sle.html"/>.
+   </para>
+  </tip>
+
+  <para>
+   With newer versions of &suseconnect;, each registered host contacts SCC or RMT automatically on a
+   daily basis. It sends unique identification of the host and description of its hardware
+   environment.
+  </para>
+ </sect1>
+ <sect1 xml:id="suseconnect-tracking-requirements">
+  <title>Requirements</title>
+
+  <para>
+   In order to use the extended &suseconnect; functionality, you need to:
+  </para>
+
+  <itemizedlist>
+   <listitem>
+    <para>
+     Run a supported and registered SLE-based host with all update patches applied.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Verify that the host system includes the <command>&suseconnect;</command> command version
+     0.3.33 or higher.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ <sect1 xml:id="suseconnect-tracking-how-it-works">
+  <title>How tracking inactive systems works</title>
+
+  <para>
+   We extended &suseconnect; with the new option <option>--keep-alive</option>. The command
+   <command>&suseconnect; --keep-alive</command> updates the last time that a host contacted SCC or
+   RMT and updates the host's hardware information.
+  </para>
+
+  <para>
+   The &suseconnect; package ships with two &systemd; units:
+  </para>
+
+  <variablelist>
+   <varlistentry>
+    <term><filename>suseconnect-keepalive.service</filename></term>
+    <listitem>
+     <para>
+      A service which runs the command <command>&suseconnect; --keep-alive</command> on demand.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><filename>suseconnect-keepalive.timer</filename></term>
+    <listitem>
+     <para>
+      A timer which runs the service <filename>suseconnect-keepalive.service</filename> once a day.
```

Comment:
I'd also add that it's done randomly. That is, it won't run this service at the same time always to avoid congestion on our side.